src/pykeen/evaluation/expectation.py

```diff
@@ -6,8 +6,6 @@
 
 import numpy as np
 
-from .metrics import ALL_TYPE_FUNCS
-
 __all__ = [
     "numeric_expected_value",
```

Comment:
do we want to include the expectation calculation directly as functions for instances of the rank based metric class, where the base class has the numeric calculation and subclasses like Hits @ K and MR can override it with closed-form implementations?